### PR TITLE
Example: aliases tasks 2 show its capabilities

### PR DESCRIPTION
**Pull request by smorin**
Example of aliases calling tasks to show its capabilities

### CODE REVIEW

File: lib/mix/lib/mix.ex [hunk 1]
```diff
@@ -161,6 +161,30 @@ defmodule Mix do
...
+          "taskalias": ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
+          "taskalias2": ["run priv/hello.exs", "run priv/hello.exs"]
+        ]
+      end
+
+  In the example above we have created 3 aliases, the first example 
+  `taskalias` will run task `hex.info`, then (run)[Mix.Tasks.Run.run/1]
+  to run a elixir script, then (cmd)[Mix.Tasks.Cmd.run/1] to run a 
+  command line bash script. This shows how powerful aliases mixed 
+  with Tasks can be.  `taskalias2` shows a limitation of tasks where
```
Comment (fertapric) [inline lib/mix/lib/mix.ex]: Double space before `taskalias2`

File: lib/mix/lib/mix.ex [hunk 2]
```diff
@@ -161,6 +161,30 @@ defmodule Mix do
   Note aliases do not show up on `mix help`.
   Aliases defined in the current project do not affect its dependencies and aliases defined in dependencies are not accessible from the current project.
 
+  Aliases can be used very powerfully to also run elixir scripts and bash commands, for example:
```
Comment (fertapric) [inline lib/mix/lib/mix.ex]: Checking the documentation I think it should be "Elixir"

File: lib/mix/lib/mix.ex [hunk 3]
```diff
@@ -161,6 +161,30 @@ defmodule Mix do
...
+      
+      #priv/hello.exs
```
Comment (fertapric) [inline lib/mix/lib/mix.ex]: An space should be included after the `#`. [Here is an example in the documentation.](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/kernel/special_forms.ex#L1118)

File: lib/mix/lib/mix.ex (hunk 2 above)
Comment (fertapric) [reply lib/mix/lib/mix.ex]: [Here is an example in the documentation](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/float.ex#L21)

File: lib/mix/lib/mix.ex [hunk 4]
```diff
@@ -161,6 +161,30 @@ defmodule Mix do
...
+      IO.puts("hello")
+
+      #priv/world.sh
+      #!/bin/sh
+      echo "world!"
+
+      defp create_aliases do
```
Comment (fertapric) [inline lib/mix/lib/mix.ex]: I would include here `# mix.exs`

File: lib/mix/lib/mix.ex (hunk 1 above)
Comment (fertapric) [inline lib/mix/lib/mix.ex]: `Tasks` could be confused with [`Task`](https://hexdocs.pm/elixir/Task.html#content). Maybe `Mix.Task`, `mix tasks` or simply `tasks`?

File: lib/mix/lib/mix.ex [hunk 5]
```diff
@@ -161,6 +161,30 @@ defmodule Mix do
...
+        [
+          "taskalias": ["hex.info", "run priv/hello.exs", "cmd priv/world.sh"],
+          "taskalias2": ["run priv/hello.exs", "run priv/hello.exs"]
+        ]
+      end
+
+  In the example above we have created 3 aliases, the first example 
+  `taskalias` will run task `hex.info`, then (run)[Mix.Tasks.Run.run/1]
+  to run a elixir script, then (cmd)[Mix.Tasks.Cmd.run/1] to run a
```
Comment (fertapric) [inline lib/mix/lib/mix.ex]: "an Elixir"

File: lib/mix/lib/mix.ex [hunk 6]
```diff
@@ -161,6 +161,30 @@ defmodule Mix do
...
+          "taskalias2": ["run priv/hello.exs", "run priv/hello.exs"]
+        ]
+      end
+
+  In the example above we have created 3 aliases, the first example 
+  `taskalias` will run task `hex.info`, then (run)[Mix.Tasks.Run.run/1]
+  to run a elixir script, then (cmd)[Mix.Tasks.Cmd.run/1] to run a 
+  command line bash script. This shows how powerful aliases mixed 
+  with Tasks can be.  `taskalias2` shows a limitation of tasks where 
+  only one of a give task will run, the second execution of `run priv/hello.exs`
```
Comment (fertapric) [inline lib/mix/lib/mix.ex]: Maybe "...shows a limitation of tasks where only one of the **given tasks** will run"?

File: lib/mix/lib/mix.ex (hunk 2 above)
Comment (fertapric) [inline lib/mix/lib/mix.ex]: I'm not sure what's the official line length for comments, but I would wrap this line (and the one above) :)

**Comment by smorin** [comment]
Can't tell why this failed anyone have ideas?

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**